snap/seed_yaml.go

```diff
@@ -58,20 +58,36 @@ type Seed struct {
 }
 
 func ReadSeedYaml(fn string) (*Seed, error) {
+	errPrefix := "cannot read seed yaml: "
+
 	yamlData, err := ioutil.ReadFile(fn)
 	if err != nil {
-		return nil, fmt.Errorf("cannot read seed yaml: %s", fn)
+		return nil, fmt.Errorf(errPrefix+"%v", err)
```

Comment:
Nitpick: looks a bit weird to concatenate errorPrefix that way when using fmt formatting anyway, why not simply `fmt.Errorf("%s%v", errorPrefix, err)` (also below)?